lib/board.rb

```diff
@@ -0,0 +1,45 @@
+class Board
+  attr_accessor :cells
+
+  def initialize(cells = nil)
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    cells.clear
+    initialize
```

Comment:
`initialize` is not typically called directly, but rather invoked via `new` when an object is instantiated.